lib/engine/game/g_1880/round/operating.rb

```diff
@@ -0,0 +1,25 @@
+# frozen_string_literal: true
+
+require_relative '../../../round/operating'
+
+module Engine
+  module Game
+    module G1880
+      module Round
+        class Operating < Engine::Round::Operating
+          def finished?
+            finished = !active_step || !any_to_act?
+            @game.end_game! if finished && trigger_game_end?
+            @current_operator = nil if finished
+            finished
+          end
+
+          def trigger_game_end?
```

Comment:
game needs to end at the end of the turn of the corp that caused the last 8 train to go away. 
Is there a better location to place this logic? or it belongs to operating.finished?

Reply:
I think its fine